src/Workspaces/Core/Desktop/Workspace/MSBuild/VisualBasic/VisualBasicProjectFileLoader.cs

```diff
@@ -880,7 +880,7 @@ public bool SetLanguageVersion(string languageVersion)
                 {
                     if (!string.IsNullOrWhiteSpace(languageVersion))
                     {
-                        _commandLineArgs.Add("/languageversion:" + languageVersion);
+                        _commandLineArgs.Add("/langversion:" + languageVersion);
```

Comment:
I think this never worked. I'm not sure if or how it would have been visible to users. 
@jasonmalinowski can probably comment on what the impact was.